src/pal/src/misc/perftrace.cpp

```diff
@@ -691,12 +691,19 @@ static
 int
 PERFWriteCounters( pal_perf_api_info * table )
 {
-    char fileName[MAX_LONGPATH];
+    char * fileName = new char[MAX_LONGPATH];
```

Comment:
I don't know before hand how much memory to allocate. Will be calling StackStringObj.OpenStringBuffer(MAX_LONGPATH) which will put it on heap. So, just creating MAX_LONGPATH on heap.